source/common/http/header_map_impl.h

```diff
@@ -54,9 +54,17 @@ class HeaderMapImpl : public HeaderMap {
   bool operator==(const HeaderMapImpl& rhs) const;
 
   // Http::HeaderMap
+  // Add without copying either key or value.€
```

Comment:
Dude, Euro are still worth more than dollars! [ ;) ]

(Fixed. Mea culpa, and thanks for the catch!)